cpp/ql/src/semmle/code/cpp/TypedefType.qll

```diff
@@ -67,26 +58,27 @@ class LocalTypedefType extends TypedefType {
  * A C++ typedef type that is directly enclosed by a class, struct or union.
  */
 class NestedTypedefType extends TypedefType {
-  NestedTypedefType() {
-    this.isMember()
-  }
+  NestedTypedefType() { this.isMember() }
 
   override string getCanonicalQLClass() { result = "NestedTypedefType" }
 
   /**
-   * DEPRECATED
+   * DEPRECATED: use `.hasSpecifier("private")` instead.
```

Comment:
Did the autoformatter write these?  Is this what we want users to do instead?

Reply:
I wrote them when fixing up the line numbering issue that made the unit tests fail. See https://github.com/Semmle/ql/pull/1870/commits/953c47e76df16062e5b9c9d89c0088e6416cbaf3. I don't know when and why these predicates were deprecated, but I thought we should follow our QLDoc style guide and provide an alternative.